core/src/main/scala/com/spotify/featran/FeatureExtractor.scala

```diff
@@ -123,11 +127,20 @@ class FeatureExtractor[M[_]: CollectionType, T] private[featran]
    * @tparam F output data type, e.g. `Array[Float]`, `Array[Double]`, `DenseVector[Float]`,
    *           `DenseVector[Double]`
    */
-  def featureValues[F: FeatureBuilder : ClassTag]: M[F] = {
+  def featureValues[F: FeatureBuilder : ClassTag]: M[F] =
+    labeledFeatureValues.map(_._2)
+
+  /**
+   * Values of the extracted features, in the same order as names in [[featureNames]] with the
+   * original raw feature object.
+   * @tparam F output data type, e.g. `Array[Float]`, `Array[Double]`, `DenseVector[Float]`,
+   *           `DenseVector[Double]`
+   */
+  def labeledFeatureValues[F: FeatureBuilder : ClassTag]: M[(T,F)] = {
```

Comment:
"Labeled" is a little misleading? I thought it meant feature labels/names without reading the code. Any better suggestion?